secret-squirrel.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
Did you intend to add `secret-squirrel.js`?

Reply:
Yup, it wouldn't let me commit without! I'm guessing those files may have been added automatically (i.e. without having npm installed so no prehooks running)